client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/ConfigurationBuilder.java

```diff
@@ -91,6 +91,7 @@ public class ConfigurationBuilder implements ConfigurationChildBuilder, Builder<
    private final List<SerializationContextInitializer> contextInitializers = new ArrayList<>();
    private final Map<String, RemoteCacheConfigurationBuilder> remoteCacheBuilders;
    private TransportFactory transportFactory = TransportFactory.DEFAULT;
+   private boolean tracingEnabled = true;
```

Comment:
Should we disable it by default (I know: API change, but still)

Reply:
I think we should enable it by default since depends also on the presence of a jar